src/client/article.ts

```diff
@@ -329,15 +329,14 @@ function richLinks(): void {
 			if (articleId) {
 				const options = {
 					headers: {
-						Accept: 'text/html',
+						Accept: 'application/json',
 					},
 				};
-				void fetch(`${articleId}?richlink`, options).then(
-					(response) => {
-						const pillar = response.headers
-							.get('pillar')
-							?.toLowerCase();
-						const image = response.headers.get('image');
+				void fetch(`${articleId}?richlink`, options)
+					.then((resp) => resp.json())
+					.then((response) => {
```

Comment:
This should force you to perform checks on the JSON data, which will in turn sidestep the `any` error.

```suggestion
					.then((response: unknown) => {
```